providers/common/kdfs/hkdf.c

```diff
@@ -14,6 +14,7 @@
 #include <openssl/evp.h>
 #include <openssl/kdf.h>
 #include <openssl/core_names.h>
+#include <openssl/params.h>
```

Comment:
Why?  This is the only change to this file.  Is there a case where it is required?
I'd not be surprised if I missed something....